cmd/operator-sdk/new/cmd.go

```diff
@@ -215,6 +221,7 @@ func doGoScaffold() error {
 
 func doAnsibleScaffold() error {
 	cfg := &input.Config{
+		Repo:           repo,
```

Comment:
Unless I'm missing something, it doesn't look like we use the repo for new ansible and helm projects. Since they aren't go projects and don't have any of the dependency manager-related files, I think we can remove these `Repo` fields for ansible and helm and update the `verifyFlags` logic such that `--repo` is only required for `migrate` when outside of `$GOPATH/src`.

Does that make sense?